src/einsteinpy/symbolic/tensor.py

```diff
@@ -42,6 +42,24 @@ def tensor(self):
         """
         return self.arr
 
+    def subs(self, *args):
```

Comment:
Name it substitute only!

Reply:
`subs` is used by sympy by default, I was just extending this function to Tensor class. It will be familiar with those using sympy

Reply:
It is like a industry standard, matlab also uses it :rofl

Reply:
Okay then